MCP server: suggest using keys from service account (#32827)
DataDog/documentation: content/en/bits_ai/mcp_server/setup/_index.md

}
{{< /code-block >}}

> For security, it is recommended to use a scoped API and application key from a [service account][23] with only the permissions you intend to use.

Reviewer: ```suggestion
For security, use a scoped API key and application key from a [service account][23] that has only the required permissions.
```
